src/config-import.js

```diff
@@ -253,18 +254,23 @@ function validatePresetsFile(presetsFile) {
   }
 }
 
-/**
- * @param {unknown} value
- * @returns {value is Record<string, unknown>}
- */
-function isRecord(value) {
-  return value !== null && typeof value === 'object' && !Array.isArray(value)
-}
-
 /**
  * @param {Record<string | symbol, unknown>} obj
  * @param {string | symbol} prop
  */
 function hasOwn(obj, prop) {
   return Object.prototype.hasOwnProperty.call(obj, prop)
 }
+
+/**
+ * @param {unknown} value
+ * @returns {Error}
+ */
+function errify(value) {
+  if (value instanceof Error) return value
+  if (typeof value === 'string') return new Error(value)
+  if (isRecord(value) && typeof value.message === 'string') {
```

Comment:
again not sure if the helper is really necessary. maybe i just prefer inlined verbosity 😄

Reply:
Simplified in 3febedcba0e512880841eee3eded8ef68072b7f0.